.vib/fluent-bit/runtime-parameters.yaml

```diff
@@ -0,0 +1,27 @@
+containerPorts:
+  http: 2378
```

Comment:
Add a test for this property

Reply:
[d884759](https://github.com/bitnami/charts/pull/15453/commits/d88475922fa9ce7829c72589eed76210a1ea9089)